src/OVAL/probes/SEAP/sexp-datatype.c

```diff
@@ -58,7 +57,9 @@ void SEXP_datatypeTbl_free(SEXP_datatypeTbl_t *t)
 
 int SEXP_datatypeTbl_init (SEXP_datatypeTbl_t *t)
 {
-        assume_d(t != NULL, -1);
+	if (t == NULL) {
+		return -1;
+	}
```

Comment:
This changes the code btw, assume_d is only turned into this when NDEBUG is not defined. Maybe you know this and are okay with it but the commit message suggests you just replaced macro with code and expect no change so I am pointing this out, **THIS IS DIFFERENT**!

Reply:
Thank you, I didn't know that. What are the reasons for checking some conditions only with debug symbols? Maybe it can improve our safety if we check some conditions always.

Reply:
Maybe the answer would be that the code is faster, but I think the difference is not so big.